docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -13,11 +13,17 @@ Model Builder uses automated machine learning (AutoML) to explore different mach
 
 You don't need machine learning expertise to use Model Builder. All you need is some data, and a problem to solve. Model Builder generates the code to add the model to your .NET application.
 
-![Model Builder Visual Studio extension user interface animation](media/ml-dotnet-model-builder.gif)
+![Model Builder Scenarios](./media/model-builder-scenarios.png)
 
 > [!NOTE]
 > Model Builder is currently in Preview.
 
+## Creating a Model Builder Project
+
+When you first start up Model Builder it will ask for you to name the project. This will create an `mbconfig` file inside of the project.
```

Comment:
```suggestion
When you first start up Model Builder it will ask for you to name the project. This will create an `mbconfig` configuration file inside of the project.
```